### PR TITLE
improve(LM rewards): uUSDrBTC-DEC week 11 liquidity mining rewards 

### PR DESCRIPTION
**Pull request by chrismaree**
This PR adds week 11's liquidity mining payouts for uUSDrBTC-DEC. The script was run from 2020-11-16 23:00:00 UTC (11271761) to 2020-11-23 23:00:00 UTC (11317321). 

**Note** that the payouts for this week's rewards are the second LM rewards who's output is based on the new [UMA Dev mining program](https://medium.com/uma-project/uma-announces-developer-mining-6f6fe15d5604). Based off the dev mining program, half of all rewards allocated to the uUSDrBTC contract, pro-rata to other contracts within the UMA dev mining program, are allocated as liquidity mining rewards. 

To see the output of this week's dev mining see PR https://github.com/UMAprotocol/protocol/pull/2216.

From this week's dev mining, a total of `20249.693071006773650943` UMA is allocated to the uUSDrBTC contract. As UMA has decided to send half of these rewards as LM rewards, the total allocated for these weekly payouts is **10124 UMA** for uUSDrBTC.

**To re-generate these results run:**
```
node ./liquidity-mining/CalculateBalancerLPRewards.js --fromBlock 11271761 --toBlock 11317321 --poolAddress="0x2dd7255b487a62d738110bd10f8bc4b4ea989778" --umaPerWeek=10124 --tokenName="yusdbtc" --week=11 --network mainnet_mnemonic
```

### CODE REVIEW

File: packages/core/scripts/liquidity-mining/yusdbtc-weekly-payouts/Week_11_Mining_Rewards.json [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    "0x0f106ff8f4dbc6f6d0144aa4a9fa88b2e67a28fc": "0.479524382273447616",
+    "0x0f9c7aae4a3db5d226371b4a68f8a47ee97fa757": "2.084855663141654512",
+    "0x0fcc9c0c92d873a8da5c51dd888b5f6e524bd128": "0.160861678863036381",
+    "0x114a4b3b0640e2b4235b0897ab343a26c97c8d14": "0.800481251248702326",
+    "0x181c2d506b4aad984e3a98f09ac8eaf895671599": "12.496081733344620373",
+    "0x1aba978cf66d4bd8ce54ab5e3aaff177ab901fb6": "0.986156627222863819",
+    "0x1b72bac3772050fdcaf468cce7e20deb3cb02d89": "117.274551102178432802",
+    "0x2208b145be1473a39e4a67c16ce75f48a074f117": "5.004548274914689482",
+    "0x25125e438b7ae0f9ae8511d83abb0f4574217c7a": "10.543516471990011887",
+    "0x25da25beb1729c137ba8ceef312fe73556b7f88d": "1877.053757375646419353",
```
Comment (nicholaspai) [inline packages/core/scripts/liquidity-mining/yusdbtc-weekly-payouts/Week_11_Mining_Rewards.json:23]: Have you sanity checked this > 1000 recipients? Just checking

**Review by nicholaspai — APPROVED** [review]
LGTM thanks for running this

File: packages/core/scripts/liquidity-mining/yusdbtc-weekly-payouts/Week_11_Mining_Rewards.json (hunk 1 above)
Comment (chrismaree) [reply packages/core/scripts/liquidity-mining/yusdbtc-weekly-payouts/Week_11_Mining_Rewards.json:23]: ye, I did check them. this is the second largest sponsor
image

**Comment by hlambur** [comment]
Rewards for uUSDrBTC-DEC send with txs: 
https://etherscan.io/tx/0x5137a9401e588180d1c928415ee6ec8c24087542768fc758e8178345883fee7a

**Pull request merged**